crates/pico_macros/src/memo.rs

```diff
@@ -14,7 +14,9 @@ use syn::{
 #[derive(Debug, FromMeta)]
 struct MemoArgs {
     #[darling(default)]
-    reference: bool,
+    inner: bool,
+    #[darling(default)]
+    inner_ref: bool,
```

Comment:
should we add a test case that exercises `inner_ref`?

Reply:
yeah, working on it. I'm not 100% convinced that it is a very useful parameter, but will see if `MemoRef` is enough.